ansible/configs/sap-integration/post_software.yml

```diff
@@ -184,58 +184,60 @@
-      become: yes
-      become_user: "{{ sap_hana_deployment_hana_sid | lower }}adm"
-      
-    - name: Ensure HANA instance is started in the secondary node
-      shell: |
-          /usr/sap/{{ sap_hana_deployment_hana_sid | upper }}/HDB{{ sap_hana_deployment_hana_instance_number }}/HDB start
-      args:
-        executable: /bin/bash
-      become: yes
-      become_user: "{{ sap_hana_deployment_hana_sid | lower }}adm"
+  - name: Ensure HANA instance is stopped in the secondary node
+    shell: |
+        /usr/sap/{{ sap_hana_deployment_hana_sid | upper }}/HDB{{ sap_hana_deployment_hana_instance_number }}/HDB stop
+    args:
+      executable: /bin/bash
+    become: true
+    become_user: "{{ sap_hana_deployment_hana_sid | lower }}adm"
+
+  - name: Ensure HANA instance is started in the secondary node
+    shell: |
+        /usr/sap/{{ sap_hana_deployment_hana_sid | upper }}/HDB{{ sap_hana_deployment_hana_instance_number }}/HDB start
+    args:
+      executable: /bin/bash
+    become: true
+    become_user: "{{ sap_hana_deployment_hana_sid | lower }}adm"
 
-    - name: Import SFLIGHT schema for demo purposes
-      shell: |
-          /usr/sap/{{ sap_hana_deployment_hana_sid | upper }}/HDB{{ sap_hana_deployment_hana_instance_number }}/exe/hdbsql \
-          -i {{ sap_hana_deployment_hana_instance_number }} -u SYSTEM -p {{ sap_hana_deployment_hana_db_system_password }} -d {{ sap_hana_deployment_hana_sid }} -m <<EOF
-          IMPORT "SFLIGHT"."SMACOURSE", \
-            "SFLIGHT"."STICKET", \
-            "SFLIGHT"."SDESSERT", \
-            "SFLIGHT"."SMEALT", \
-            "SFLIGHT"."SCURX", \
-            "SFLIGHT"."STRAVELAG", \
-            "SFLIGHT"."SFLIMEAL", \
-            "SFLIGHT"."SFLIGHT", \
-            "SFLIGHT"."SCPLANE", \
-            "SFLIGHT"."SNVOICE", \
-            "SFLIGHT"."SPPLANE", \
-            "SFLIGHT"."SMEAL", \
-            "SFLIGHT"."SCITAIRP", \
-            "SFLIGHT"."SCARPLAN", \
-            "SFLIGHT"."SCARR", \
-            "SFLIGHT"."SCOUNTER", \
-            "SFLIGHT"."SPFLI", \
-            "SFLIGHT"."SAIRPORT", \
-            "SFLIGHT"."SBUSPART", \
-            "SFLIGHT"."SAPLANE", \
-            "SFLIGHT"."SMENU", \
-            "SFLIGHT"."SSTARTER", \
-            "SFLIGHT"."SCURR", \
-            "SFLIGHT"."SGEOCITY", \
-            "SFLIGHT"."SCUSTOM", \
-            "SFLIGHT"."SBOOK" \
-          FROM '/software/SFLIGHT' WITH REPLACE THREADS 8;
-          EOF
-      args:
-        executable: /bin/bash
-      become: yes
-      become_user: "{{ sap_hana_deployment_hana_sid | lower }}adm"
+  - name: Import SFLIGHT schema for demo purposes
+    # yamllint disable rule:line-length
+    shell: |
+        /usr/sap/{{ sap_hana_deployment_hana_sid | upper }}/HDB{{ sap_hana_deployment_hana_instance_number }}/exe/hdbsql \
+        -i {{ sap_hana_deployment_hana_instance_number }} -u SYSTEM -p {{ sap_hana_deployment_hana_db_system_password }} -d {{ sap_hana_deployment_hana_sid }} -m <<EOF
```

Comment:
This line length issue is not resolved... and the use of `|` block-quote with all these backslashes should be fixed.